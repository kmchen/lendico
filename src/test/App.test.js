import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { AppComponent } from '../App';

configure({ adapter: new Adapter() });

describe('App component', () => {
  let appComponent;

  beforeEach(() => {
    const props = {
      dispatch: jest.fn(),
      monthlyInstallment: "1",
      duration: "2",
      amount: "3",
    };
    appComponent = shallow(<AppComponent {...props} />);
  });

  it('should render App component', () => {
    expect(appComponent).toMatchSnapshot();
  });
});
