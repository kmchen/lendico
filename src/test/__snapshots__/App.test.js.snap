// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render App component 1`] = `
<div
  className="makeStyles-root-1"
>
  <CssBaseline />
  <NavBar />
  <main>
    <SearchBar
      amountChange={[Function]}
      durationChange={[Function]}
      getMonthlyInstallment={[Function]}
    />
    <Result
      monthlyInstallment="1"
    />
  </main>
  <Footer />
</div>
`;
